providers/keycloak/src/airflow/providers/keycloak/auth_manager/services/token.py

```diff
@@ -51,3 +51,47 @@ def create_token_for(
     )
 
     return get_auth_manager().generate_jwt(user, expiration_time_in_seconds=expiration_time_in_seconds)
+
+
+def create_token_for_client_credentials(
```

Comment:
Reads better (to me)

```suggestion
def create_client_credentials_token(
```